### DISCUSSION TITLE
Opa eval pass env variables

### DISCUSSION THREAD

**Discussion by kotyara85** [open-policy-agent/community#248] [OPA and Rego]
Hello everyone,
Is there a way to pass env variables require in regos during opa eval run?
I tried to run like this but it did work
```
SERVERS=https://example.com opa eval --input=../input.json --format=pretty --profile --data=./policies/ --count=1 'data.allow'
```

Thanks

**Comment by anderseknert** [comment]
Hi @kotyara85 👋 

If you want to read environment variables inside of a policy, you can use the `opa.runtime` built-in function for that purpose; e.g.

```shell
$ MY_VAR=foo opa eval -f pretty 'opa.runtime().env.MY_VAR'
"foo"
```
